src/game/Box.js

```diff
@@ -20,18 +20,21 @@ export default class Box extends Component {
         }
     };
 
-    render(boxSize) {
+    render(boxSize, label = 'box') {
         const { enabled, boxValue } = this.props;
         const pieceImageArray = initImagesRequire();
 
         return (
-            <TouchableHighlight
-                style={[boxSize, styles.box, enabled ? styles.enabled : '']}
-                onPress={this.handlePress}
-                underlayColor={clickedBoxColor}
-            >
-                <Image style={boxSize} source={pieceImageArray[boxValue]} />
-            </TouchableHighlight>
+            <label>
+                {label}
+                <TouchableHighlight
```

Comment:
https://facebook.github.io/react-native/docs/accessibility.html

```
accessible={true}
accessibilityLabel={'Tap me!'}
```

Unfortunately, `react-testing-library` won't be able to find an element by its `accessibilityLabel`. I guess you'll have to add a `data-testid` attribute in addition to the accessibility attributes (good practice) and use `getByTestId` instead of adding those ugly labels